setup.rst

```diff
@@ -223,68 +124,64 @@ complexity of your project):
     $ cd my_project_name/
     $ composer update
 
-.. tip::
-
-    Symfony provides a command to check whether your project's dependencies
-    contain any known security vulnerability:
+.. _install-existing-app:
 
-    .. code-block:: terminal
+Installing an Existing Symfony Application
+------------------------------------------
 
-        $ php bin/console security:check
+When working collaboratively in a Symfony application, it's uncommon to create
+a new Symfony application as explained in the previous sections. Instead,
+someone else has already created and submitted it to a shared repository.
 
-    A good security practice is to execute this command regularly to be able to
-    update or replace compromised dependencies as soon as possible.
+It's recommended to not submit some files (``.env``) and directories (``vendor/``,
+cache, logs) to the repository, so you'll have to do the following when
+installing an existing Symfony application:
```

Comment:
Yes ... we'll do that in the upcoming reference article about configuring Symfony apps: #7386